### PR TITLE
NISRA - Validation + Default Routing

### PR DESCRIPTION
**Pull request by MebinAbraham**
### What is the context of this PR?
Added validation and default routing to NISRA.

### How to review 
Ensure:
- Changes are as per instructions on Trello.

[Quick launch for NIR schema](https://v3-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&case_type=HI&region_code=GB-NIR&url=https://raw.githubusercontent.com/ONSdigital/eq-survey-runner/eq-3163-default-routing-validation-nisra/data/en/census_individual_gb_nir.json)

### CODE REVIEW

**Comment by MebinAbraham** [comment]
[Default routing & Validation (NISRA)](https://trello.com/c/E0wlPBqC/3163-default-routing-validation-nisra)

File: data-source/jsonnet/northern-ireland/blocks/individual/identity-and-health/religion.jsonnet
Comment (berroar) [inline data-source/jsonnet/northern-ireland/blocks/individual/identity-and-health/religion.jsonnet]: Is this a typo?

Comment (MebinAbraham) [reply data-source/jsonnet/northern-ireland/blocks/individual/identity-and-health/religion.jsonnet]: Good spot. Fixed.

**Review by donnap573 — COMMENTED** [review]
Can you understand, speak, read or write Irish shouldn't be mandatory
Can you understand, speak, read or write Ulster Scots shouldn't be mandatory
Do you have any of the following physical conditions lasting or expected to last 12 months or more? shouldn't be mandatory
If degree, GCSE, A Level and NVQ questions are all skipped, all no/none or a combination of skipped and no/none you should be routed to the other qualifications question

**Comment by MebinAbraham** [comment]
> Can you understand, speak, read or write Irish shouldn't be mandatory
> Can you understand, speak, read or write Ulster Scots shouldn't be mandatory
> Do you have any of the following physical conditions lasting or expected to last 12 months or more? shouldn't be mandatory
> If degree, GCSE, A Level and NVQ questions are all skipped, all no/none or a combination of skipped and no/none you should be routed to the other qualifications question

Have fixed the mandatory/optional issue. 

With regards to qualifications, I didn't initially make the change as I thought it was already done.
I have now added routing for this. Also does this need to be same on ENG/WLS because the routing on ENG/WALES is if they have said no to all of them, instead of skipped aswell.

**Review by berroar — APPROVED** [review]
Follows default routing and validation listed in the spreadsheet

**Review by donnap573 — APPROVED** [review]

File: data/en/census_individual_gb_nir.json
Comment (bitdivision) [inline data/en/census_individual_gb_nir.json]: Why is this being committed?

**Review by bitdivision — CHANGES_REQUESTED** [review]
Changes all match up with card / s'sheet.

Will approve once the built schema has been removed.

File: data/en/census_individual_gb_nir.json
Comment (MebinAbraham) [reply data/en/census_individual_gb_nir.json]: For quick launch for Donna. That how it was being done for all the content changes. They are removed once approved.

Comment (bitdivision) [reply data/en/census_individual_gb_nir.json]: I see. We should sort out a proper solution for that...

**Review by bitdivision — APPROVED** [review]

**Comment by bitdivision** [comment]
@MebinAbraham I'm guessing you reran travis which is why codacy isn't running? Might just need a `git commit --amend; git push -f`

**Pull request merged**
